source/community/release_notes/palm.rst

```diff
@@ -15,30 +15,220 @@ These are the release notes for the Palm release, the 16th community release of
+* `learner_home_mfe.enable_learner_home_amplitude_recommendations <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/lms/djangoapps/learner_home/recommendations/waffle.py#L10>`_
+* `learner_recommendations.enable_course_about_page_recommendations <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/lms/djangoapps/learner_recommendations/toggles.py#L10>`_
+* `learner_recommendations.enable_dashboard_recommendations <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/lms/djangoapps/learner_recommendations/toggles.py#L24>`_
+* `student.enable_enrollment_confirmation_email <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/common/djangoapps/student/toggles.py#L61>`_
+* `student.enable_fallback_recommendations <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/common/djangoapps/student/toggles.py#L26>`_
+* `third_party_auth.apple_user_migration <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/common/djangoapps/third_party_auth/toggles.py#L9>`_
+* `video_config.public_video_share <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/openedx/core/djangoapps/video_config/toggles.py#L8>`_
+
+The following Features/Toggles were removed:
+
+* `block_structure.raise_error_when_not_found <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/openedx/core/djangoapps/content/block_structure/config/__init__.py#L51>`_
+* `credentials.use_learner_record_mfe <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/openedx/core/djangoapps/credentials/config.py#L10>`_
+
+The following new settings were added:
+
+* `SECURITY_PAGE_URL <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/lms/envs/common.py#L3355)>`_
+
+The following settings were removed:
+
+* `LEARNER_RECORD_MFE_URL <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/lms/envs/common.py#L4894>`_
+* `XBLOCK_SELECT_FUNCTION <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/lms/envs/common.py#L1531>`_
+
+
 
 Deprecations & Removals
 ***********************
 
+The Django setting :code:`JWT_AUTH.JWT_PRIVATE_SIGNING_JWK` can be removed from CMS configs if it is still present, as it has only been used by the LMS for some time now.
+
 
 Developer Experience
 ********************
 
+API changes
+===========
+
+* Filter by active courses in the course listing API.
+* Added :code:`certificate_available_date` to course detail API.
+
+New filters
+===========
+
+* Added :code:`VerticalBlockRenderCompleted` filter that will be called after a VerticalBlock's rendering is completed.
```

Comment:
Also this one that was added in the same PR as `VerticalBlockRenderCompleted`: https://github.com/openedx/edx-platform/pull/31388/files#diff-33588454a586bfbea6bd37f51652b19fbb4e77785791c9de43eaa68c7b527788R120-R128
```suggestion
* Added :code:`VerticalBlockRenderCompleted` filter that will be called after a VerticalBlock's rendering is completed.
* Added :code:`VerticalBlockChildRenderStarted` that will be called when every child block of a VericalBlock is about to be rendered.
```